src/components/CompareResults/ResultsMain.tsx

```diff
@@ -29,8 +30,11 @@ function ResultsMain() {
 
   const themeMode = useAppSelector((state) => state.theme.mode);
   const [searchParams, setSearchParams] = useSearchParams();
-  const [searchTerm, setSearchTerm] = useState('');
 
+  // This is our custom hook that updates the search params without a rerender.
+  const [rawSearchParams, updateSearchParams] = useRawSearchParams();
+  const initialSearchTerm = rawSearchParams.get('search') ?? '';
+  const [searchTerm, setSearchTerm] = useState(initialSearchTerm);
```

Comment:
can you also do the change in SubtestsResultsMain too?

Reply:
Ah right I forgot about that page. Fixed it now.